apps/web/src/app/[orgShortcode]/_components/sidebar-content.tsx

```diff
@@ -82,30 +82,37 @@ function OrgMenu() {
           className={
             'bg-base-1 border-base-5 hover:bg-base-2 flex w-full flex-row items-center justify-between gap-2 rounded-lg border p-3 shadow-sm'
           }>
-          <div className={'flex flex-row items-center gap-2'}>
+          <div className={'flex w-full flex-row items-center gap-2'}>
             <Avatar
               avatarProfilePublicId={currentOrg.publicId}
               avatarTimestamp={currentOrg.avatarTimestamp}
               name={currentOrg.name}
               size="lg"
               hideTooltip
             />
-            <div className={'flex flex-col items-start justify-start'}>
-              <span className={cn('text-sm font-semibold leading-none')}>
+            <div
+              className={
+                'flex w-full flex-col items-start justify-start gap-1 overflow-hidden'
+              }>
+              <span
+                className={cn(
+                  'block w-full truncate text-left text-sm font-semibold leading-none'
+                )}>
                 {currentOrg.name}
               </span>
               <span
-                className={cn('text-base-11 text-xs font-medium leading-none')}>
+                className={cn(
+                  'text-base-11 block w-full truncate text-left text-xs font-medium leading-none'
+                )}>
                 {displayName}
               </span>
             </div>
-          </div>
-
-          <div className={'h-5 w-5 p-0.5'}>
-            <CaretUpDown className={'h-4 w-4'} />
+            <div className={'h-5 w-5 p-0.5 pl-1'}>
+              <CaretUpDown className={'h-4 w-4'} />
+            </div>
           </div>
         </DropdownMenuTrigger>
-        <DropdownMenuContent className="bg-base-1 border-base-5 flex w-fit flex-col gap-0 p-0">
+        <DropdownMenuContent className="bg-base-1 border-base-5 flex w-[350px] flex-col gap-0 p-0">
```

Comment:
make it `w-fit max-w-[250px]` that would look best from my experiments

Reply:
done!